### PR TITLE
[luci] Negative tests for CircleAbs IR

### PR DESCRIPTION
**Pull request by seanshpark**
This will add some negative tests for CircleAbs IR

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #1623, from draft #2086

File: compiler/luci/lang/src/Nodes/CircleAbs.test.cpp [hunk 1]
```diff
@@ -29,3 +32,63 @@ TEST(CircleAbsTest, constructor)
...
+  abs_node.x(nullptr);
+  ASSERT_EQ(nullptr, abs_node.x());
+}
+
+TEST(CircleAbsTest, arity_NEG)
+{
+  luci::CircleAbs abs_node;
+
+  ASSERT_NO_THROW(abs_node.arg(0));
+  ASSERT_THROW(abs_node.arg(1), std::out_of_range);
```
Comment (llFreetimell) [inline compiler/luci/lang/src/Nodes/CircleAbs.test.cpp:69]: I am not sure `std::out_of_range` should come first or not like `ASSERT_EQ(nullptr, abs_node.x());`.
If this is right, ignore this comment :)

Comment (seanshpark) [reply compiler/luci/lang/src/Nodes/CircleAbs.test.cpp:69]: From gtest,
```cxx
#define ASSERT_THROW(statement, expected_exception)
```
throw is not like others... strange :)

Comment (llFreetimell) [reply compiler/luci/lang/src/Nodes/CircleAbs.test.cpp:69]: @seanshpark Then my comment was wrong :) Thank you for detailed information!

**Pull request merged**